shared/chat/container.js

```diff
@@ -1,23 +1,7 @@
 // @flow
-import React, {Component} from 'react'
-import {connect} from 'react-redux'
 import Chat from './index'
-
-import {loadInbox} from '../actions/chat'
-
-class ChatContainer extends Component {
-  componentWillMount () {
-    this.props.loadInbox()
-  }
-
-  render () {
-    return <Chat />
-  }
-}
+import {connect} from 'react-redux'
 
 export default connect(
   (state: any) => ({}),
-  (dispatch: Dispatch) => ({
-    loadInbox: () => dispatch(loadInbox()),
-  })
-)(ChatContainer)
+)(Chat)
```

Comment:
Maybe not but i didn't want to make that change until it was clear. i could imagine we might need some thing (like a modal or something) that needs it. seems harmless like this for now